debian/rules

```diff
@@ -49,6 +51,7 @@ override_dh_shlibdeps:
 	    -X site-packages/PIL/.libs \
 	    -X site-packages/Pillow.libs \
 	    -X site-packages/psycopg2
+        # TODO: Do we need to add our rust package here?
```

Comment:
Not sure about this one - I'm not familiar enough with debian packages to answer confidently. But since the CI job is passing it should be fine?

Reply:
> But since the CI job is passing it should be fine?

Oh actually it doesn't look like we're testing building the debs in CI? (or the job has been renamed and I can't find it on this PR's status)

Reply:
Oh gah, I've broken the `github/workflows/release-artifacts.yml` workflow :facepalm:

Reply:
Ah that'd explain it. It's annoying GitHub doesn't show a warning or error on the PR when that's the case.

Reply:
Empirically the debs seem to work when I tested them, and the build script does run the unit tests, so I think we should be good.